WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -47,6 +47,17 @@
 static NSTimeInterval NotificationsUndoTimeout          = 4;
 static NSString const *NotificationsNetworkStatusKey    = @"network_status";
 
+static CGFloat const RatingsViewHeight                  = 100.0f;
```

Comment:
Super nitpicky but, maybe drop the `f` from the various `CGFloat`s and let the compiler worry about assigning the appropriate floating point flavor.